src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -44,14 +44,15 @@ def _get_login_token(login_server, only_refresh_token=True, repository=None):
 
     from azure.cli.core._profile import Profile
     profile = Profile()
-    sp_id, refresh, tenant = profile.get_refresh_token()
+    sp_id, refresh, access, tenant = profile.get_refresh_token()
 
     headers = {'Content-Type': 'application/x-www-form-urlencoded'}
     if sp_id is None:
         content = {
-            'grant_type': 'refresh_token',
+            'grant_type': 'access_token_refresh_token',
             'service': params['service'],
             'tenant': tenant,
+            'access_token': access,
```

Comment:
Do we need to send both access and refresh tokens or just either one of them? Just some NIT comments: If it is either one of them, can we make `grant_type` explicitly `access_token` or `refresh_token` so it would be clear what I am sending and what I will receive back? Then we can have one token field which can be access or refresh token.